services/website-service/service.yaml

```diff
@@ -136,3 +141,110 @@ Resources:
                         }]
                     }
 
+    AutoScalingRole:
+        Type: AWS::IAM::Role
+        Properties:
+            AssumeRolePolicyDocument:
+                Version: '2012-10-17'
+                Statement:
+                    Action:
+                    - 'sts:AssumeRole'
+                    Effect: Allow
+                    Principal:
+                        Service:
+                        - application-autoscaling.amazonaws.com
+            Path: /
+            Policies:
+            - PolicyName: ecs-service-autoscaling
+              PolicyDocument:
+                  Statement:
+                      Effect: Allow
+                      Action:
+                      - application-autoscaling:*
+                      - cloudwatch:DescribeAlarms
+                      - cloudwatch:PutMetricAlarm
+                      - ecs:DescribeServices
+                      - ecs:UpdateService
+                      Resource: "*"
```

Comment:
shouldn't the resource be more specific to only allow operations for the current stack?

Reply:
Hi @wichert, would you suggest adding a policy condition to only apply to resources tagged with the cloudformation stack id? I haven't seen this type of condition used around though.